components/object-property-list/screen-reader-pause.js

```diff
@@ -0,0 +1,15 @@
+import { html, LitElement } from 'lit';
+import { getSeparator } from '@brightspace-ui/intl/lib/list.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+
+export class ScreenReaderPause extends LitElement {
```

Comment:
Did you intend `/object-property-list/` to be the home for this new component?

Reply:
Yes I did, mostly because it's only used by the OPL (currently) and I didn't have a better place to put it. Do you have a suggestion? Could go in `helpers`, though those are mostly not web-components...

Reply:
Maybe in the `components/offscreen` folder?

Reply:
That might be an ok place. Not sure where else it would go unless it goes into a dedicated folder. I think it will seem weird for anyone else to reference it in its current location. Maybe others have an opinion. 🤷

Reply:
Okay, moved it to `offscreen` and added some documentation 👍. And I've added it to the list of imports on the `documentation` issue.